### DISCUSSION TITLE
raycast bash script

### DISCUSSION THREAD

**Discussion by lexilexikon** [danielmiessler/Fabric#1032] [Q&A] [answered]
I've installed fabric and tried to use the Raycast script command to get the transcript of a YouTube video. The bash script is breaking and it keeps saying 'invalid YouTube URL, can't get video ID'. That was the URL I retrieved: https://youtu.be/7Inkwc23U1M. I've also installed a YouTube API.

Can anyone help me figure out what's going wrong?"

**Comment by naro110** [comment, marked as answer]
change argument to
# @raycast.argument1 { "type": "text", "placeholder": "Input text", "optional": false, "percentEncoded": false}
this worked for me

**Reply by lexilexikon** [reply]
Thank you! Now it works like a charm.
